src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-28">
+      <div class="calendar__block calendar__block--start-day-mon"></div>
```

Comment:
```suggestion
    <div class="calendar calendar--month-length-28 calenda--start-day--mon">
      <div class="calendar__block"></div>
```
this modifier should be on the whole calendar block (use first-child - all your logic is correct)